device/ipv4.yaml

```diff
@@ -34,6 +34,8 @@ components:
             The IPv4 address of the gateway
           type: string
           format: ipv4
+        manual_gateway_mac: 
```

Comment:
Does configured_gateway_mac sound better ?

Reply:
`manual_gateway_mac` is also fine I think. @ajbalogh @ankur-sheth @ashutshkumr Can you please comment which one would be more appropriate to go with `manual_gateway_mac` or `configured_gateway_mac` ?

Reply:
Here's what I suggest.

```python
 ip1.gateway_mac.learned = False
 ip1.gateway_mac.address = "aa:aa:aa:aa:aa:aa"
```

Reply:
actually a better approach would be `auto` vs `value` choice.

```python
# manual config
ip1.gateway_mac.value = "aa:aa:aa:aa:aa:aa"

# auto config
nothing required
```